.rubocop.yml

```diff
@@ -20,6 +20,9 @@ Metrics/MethodLength:
   # Change the default 10 lines limit value
   Max: 50
 
+Metrics/ModuleLength:
```

Comment:
Avoid this if possible, try to split your modules

Reply:
Removed